scripts/tweak_pipeline.pl

```diff
@@ -65,9 +66,13 @@ sub main {
     } else {
         die "\nERROR: Connection parameters (url or reg_conf+reg_alias) need to be specified\n";
     }
-
     if(@$tweaks) {
-        my $need_write = $pipeline->apply_tweaks( $tweaks );
+        my ($need_write, $listRef, $responceStructure) = $pipeline->apply_tweaks( $tweaks );
```

Comment:
Same comment here as in Scripts/InitPipeline.pm: snake case is the norm. For consistency use the same variable names in both files. However, `$json` would clash with the instance of the JSON codec below, so rename `$json` in one of the two files